src/Application/Migrations/17_function_ticket_dependency_satisfied.sql

```diff
@@ -27,11 +27,20 @@ BEGIN
 		t.id = param_ticket_id;
 
 	SELECT
-		state >= p.dependent_ticket_trigger_state INTO satisfaction
+		dependent_ticket_state.sort >= configured_trigger_state.sort INTO satisfaction
 	FROM
 		tbl_ticket t
 	JOIN
 		tbl_project p ON t.project_id = p.id
+	JOIN
+		-- if given ticket is no encoding ticket, function will return NULL
+		tbl_ticket_state dependent_ticket_state ON
+			t.ticket_type = 'encoding' AND
+			dependent_ticket_state.ticket_state = t.ticket_state
+	JOIN
+		tbl_ticket_state configured_trigger_state ON
+			t.ticket_type = 'encoding' AND
+			configured_trigger_state.ticket_state = p.dependent_ticket_trigger_state
```

Comment:
> or just use "master ticket", whenever you mean the master ticket.

I'm probably the reason we're not currently doing this. As we set `depends_on` for encoding profiles, we have no notion of a _master_ or an informal _subticket_ anywhere else in the UI or code. Therefor i suggested something like _dependent_. I like _dependee_ and maybe _dependent_? The last is a little more common. This would also free us from the language issues around _master/slave_.

Reply:
>  in #171 it's about to be called that, if I saw that correctly.

I think this is only a comment and a UI label – I already requested changing this some time ago.

Reply:
The semantical change of the parameter is likely an error, I will investigate that.